vnext/ReactWindows-Universal.sln

```diff
@@ -300,11 +303,24 @@ Global
 		{6C60E295-C8CA-4DC5-B8BE-09888F58B249}.Debug|x64.ActiveCfg = Debug|x64
 		{6C60E295-C8CA-4DC5-B8BE-09888F58B249}.Debug|x64.Build.0 = Debug|x64
 		{6C60E295-C8CA-4DC5-B8BE-09888F58B249}.Debug|x86.ActiveCfg = Debug|Win32
+		{6C60E295-C8CA-4DC5-B8BE-09888F58B249}.Debug|x86.Build.0 = Debug|Win32
 		{6C60E295-C8CA-4DC5-B8BE-09888F58B249}.Release|ARM.ActiveCfg = Release|Win32
 		{6C60E295-C8CA-4DC5-B8BE-09888F58B249}.Release|ARM64.ActiveCfg = Release|Win32
 		{6C60E295-C8CA-4DC5-B8BE-09888F58B249}.Release|x64.ActiveCfg = Release|x64
 		{6C60E295-C8CA-4DC5-B8BE-09888F58B249}.Release|x64.Build.0 = Release|x64
 		{6C60E295-C8CA-4DC5-B8BE-09888F58B249}.Release|x86.ActiveCfg = Release|Win32
+		{1958CEAA-FBE0-44E3-8A99-90AD85531FFE}.Debug|ARM.ActiveCfg = Debug|Win32
+		{1958CEAA-FBE0-44E3-8A99-90AD85531FFE}.Debug|ARM64.ActiveCfg = Debug|Win32
+		{1958CEAA-FBE0-44E3-8A99-90AD85531FFE}.Debug|x64.ActiveCfg = Debug|x64
+		{1958CEAA-FBE0-44E3-8A99-90AD85531FFE}.Debug|x64.Build.0 = Debug|x64
+		{1958CEAA-FBE0-44E3-8A99-90AD85531FFE}.Debug|x86.ActiveCfg = Debug|Win32
+		{1958CEAA-FBE0-44E3-8A99-90AD85531FFE}.Debug|x86.Build.0 = Debug|Win32
+		{1958CEAA-FBE0-44E3-8A99-90AD85531FFE}.Release|ARM.ActiveCfg = Release|Win32
+		{1958CEAA-FBE0-44E3-8A99-90AD85531FFE}.Release|ARM64.ActiveCfg = Release|Win32
```

Comment:
I don't think this is right?

Reply:
We don't actually support ARM64 since we are on RS2 min and ARM62 needs RS3. #4031

Reply:
FYI I seem to remember there was a recent change that made the min RS3, please double check :)

---
In reply to: [379610035](https://github.com/microsoft/react-native-windows/pull/4101#discussion_r379610035) [](ancestors = 379610035)